mm2src/adex_cli/src/adex_config.rs

```diff
@@ -151,7 +154,12 @@ impl AdexConfigImpl {
         let adex_path_str = cfg_path
             .to_str()
             .ok_or_else(|| error_anyhow!("Failed to get cfg_path as str"))?;
-        rewrite_json_file(self, adex_path_str)
+        rewrite_json_file(self, adex_path_str)?;
```

Comment:
is this advised? Wouldn't it be better/safer to assume config file has correct permission setting as opposed to implementing a de-facto "chmod" into mm2? 
Rly concerned over likely unneeded filestream ops and speaking general - against this.